ecommerce/frontend/src/components/pages/LoginPage.less

```diff
@@ -0,0 +1,34 @@
+.loginForm {
+    display: block;
+    margin-top: 80px;
+    margin-left: auto;
+    margin-right: auto;
+    width: 40%;
```

Comment:
I think, letting the component enlarge as much as possible is a better approach. To restrict the growth, we can pass the width prop from the parent to fix its area.

Reply:
Yes please remove the width: property to allow the component to take as much space available of the parent component.